src/pages/merge/IndividualsMergePending.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
```

Comment:
maybe a few too many components in one file here, otherwise lgtm

Reply:
Agreed. This file feels ever-so-slightly long and might could use some chopping.

Reply:
Reduced components in https://github.com/WildMeOrg/codex-frontend/pull/306/commits/83eeaef18db77413cc4e8b62bb7a5d5ffac60fa7